qiskit/algorithms/__init__.py

```diff
@@ -128,8 +128,8 @@
     VarQRTE
     PVQD
     PVQDResult
-    EvolutionResult
-    EvolutionProblem
+    TimeEvolutionResult
+    TimeEvolutionProblem
```

Comment:
This is the former now pending deprecated evolvers logic right. The EvolutionResult/Problem should stay there as-is, until they are deprecated/removed. TimeEvolutionResult/Problem is in the Time Evolvers section below and should not be here. I think this change ca be simply reverted.

What is needed is to move PVQD and PVQDResult down to the Time Evolvers section from here the same way PVQD was moved across folders. All that should be there is what was released to stable ie
```
    RealEvolver
    ImaginaryEvolver
    TrotterQRTE
    EvolutionResult
    EvolutionProblem
```
VarQITE and VarQRTE were removed by #8750 right - I see this branch is out-of-date so maybe that has not yet been incorporated it seems.

Reply:
Yes, sorry, I was not updating the branch and now fixed these things!